turnerlabs/yt_push
# DRACO-3608 Set a given taxonomy field as the tags on an uploaded video

Lets you configure a tags field to be the value of the YouTube tags when the video is uploaded.

To test:

* Create a new YouTube media type using the Drush command: `drush cay`.
* Authenticate that media type.
* Create a new YouTube media with some tags and check that the tags were reflected in the uploaded video.
